include/vcpkg/base/files.h

```diff
@@ -142,9 +142,14 @@ namespace vcpkg
 
     struct WriteFilePointer : FilePointer
     {
+        enum class Append
+        {
+            YES,
+            NO,
+        };
```

Comment:
```suggestion
        enum class Append
        {
            NO = 0,
            YES,
        };
```
still needs this to be consistent.